Extension/lib/content-script/preload.js

```diff
@@ -126,6 +127,16 @@
     };
 
     /**
+     * Checks location's protocol.
+     * Returns true if we are on a 'http(s)' or 'about:' page

```

Comment:
It makes sense to have a better description of why do we need it on about: page.

This is due to the circumvention with using iframe's contentWindow